tests/vmi_networking_test.go

```diff
@@ -722,42 +722,46 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 			Expect(len(serverVMI.Status.Interfaces)).To(Equal(1))
 
-			serverIP := serverVMI.Status.Interfaces[0].IP
-			if netutils.IsIPv6String(serverIP) {
-				By("Checking traceroute from vmi to cluster nodes gateway")
-				// Cluster nodes subnet (docker network gateway)
-				// Docker network subnet cidr definition https://bit.ly/2wZTgMK
-				err := tests.CheckForTextExpecter(serverVMI, createExpectTraceroute6("2001:db8:1::1"), 30)
-				Expect(err).ToNot(HaveOccurred(), "Failed to traceroute to VMI %s within the given timeout", serverVMI.Name)
-			} else {
-				if ipv4NetworkCIDR == "" {
-					ipv4NetworkCIDR = api.DefaultVMCIDR
+			for _, serverIP := range serverVMI.Status.Interfaces[0].IPs {
```

Comment:
Please add `By(IPv4)` and `By(IPv6)`.

Reply:
added already after it checks the type on lines
https://github.com/kubevirt/kubevirt/pull/3989/files#diff-f911024a391a99cfdef2241f409a4748R731
https://github.com/kubevirt/kubevirt/pull/3989/files#diff-f911024a391a99cfdef2241f409a4748R740

will see if more needed in the beginning of each section

Reply:
done